### ISSUE TITLE
Blue Mage: Refueling does not grant haste when cast, works on zoning

### ISSUE THREAD

**Issue by Jimmayus** [EdenServer/community#2100]
### Checklist

- [X] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [X ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [X] I have provided my client version in the details. (type /ver into your game window)

### Details

Wikipedia: https://ffxiclopedia.fandom.com/wiki/Refueling

Per the above link, Refueling should grant the Blue Mage (and, when you use the diffusion buff, to all party members) a 10% haste effect. Occasionally Refueling will be bugged and be non functional until zoning. The effect looks like this:

![Jimmayus_2019 10 06_105127](https://user-images.githubusercontent.com/45926115/66271878-c1d7ec00-e828-11e9-97eb-298b3e6dab65.png)

Upon zoning to Qufim it worked again:

![Jimmayus_2019 10 06_110025](https://user-images.githubusercontent.com/45926115/66271882-cac8bd80-e828-11e9-976d-02e0786cb3d6.png)

I forgot to screenshot that I tested utsusemi recasts to confirm the haste % was missing, but as you can see from comparing the two screenshots in the above the game says I "received the effect of KO (lol)" and no haste icon appeared on my buff bar. In the below picture everything is normal including recasts (not pictured).

**Comment by Jimmayus** [comment]
I zoned back into Behemoth's and received the bug again. Zoning back into Qufim and then rezoning into Behemoth's bugged me again.

**Comment by Jimmayus** [comment]
tl;dr looks like maybe it's individual zones doing this

On a hunch I thought it might be all dungeon zones, but that does not appear to be the case. See the below two pictures:

![Jimmayus_2019 10 06_130639](https://user-images.githubusercontent.com/45926115/66273530-657dc800-e83a-11e9-9dd0-a02ddf09fc0f.png)

In this one I cast and received the haste buff in the chateau. In the second picture:

![Jimmayus_2019 10 06_130719](https://user-images.githubusercontent.com/45926115/66273542-775f6b00-e83a-11e9-95b1-b4d0be351c63.png)

I cast and received the haste buff in Bostanieux, which I believe is a dungeon area (honestly not sure if there is a category of zones like this but is seemed plausible). Anyway, it looks to me like this is not a categorical bug, but rather one that is appearing consistently in only certain zones.

**Comment by KnowOne134** [comment]
Well, you provided alot of testing. The type of zone won't matter. I'll take a look at it soon. When it doesnt work do you always get the effect of KO message?

**Comment by Jimmayus** [comment]
Yes, it is consistently a "received the effect of KO" message.

**Comment by KnowOne134** [comment]
closing due to not being able to reproduce and no other mentions of it happening
